Rubberduck.VBEEditor/VBEHost/CorelDRAWApp.cs

```diff
@@ -0,0 +1,17 @@
+using Corel.GraphicsSuite.Interop.CorelDRAW;
+
+namespace Rubberduck.VBEditor.VBEHost
+{
+    public class CorelDRAWApp : HostApplicationBase<Application>
+    {
+        public CorelDRAWApp() : base("CorelDRAW") { }
+
+		//Function RunMacro(ModuleName As String, MacroName As String, Parameter() As Variant) As Variant
+		//Where ModuleName appears to mean ProjectName, and MacroName has to be provided as "ModuleName.ProcName"
+		
+        public override void Run(QualifiedMemberName qualifiedMemberName)
+        {
+            Application.GMSManager.RunMacro(qualifiedMemberName.QualifiedModuleName.ProjectName.ToString(), qualifiedMemberName.ToString(), null);
```

Comment:
Also the `ProjectName` is already a string, the `ToString` call is redundant, and calling `ToString` on the qualifiedMemberName would return the *qualified* member name, like `Proj.TestModule1.TestMethod1`, which doesn't seem like what the API is expecting.  Would this work?

    Application.GMSManager.RunMacro(qualifiedMemberName.QualifiedModuleName.ProjectName, qualifiedMemberName.MemberName);